test/functional/feature_nulldummy.py

```diff
@@ -15,7 +15,7 @@
 
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import *
-from test_framework.mininode import CTransaction, network_thread_start
+from test_framework.mininode import CTransaction
```

Comment:
Can be changed to `from test_framework.messages import CTransaction` :)

Reply:
;)